sdk/php/src/ValueObject/DaggerFunction.php

```diff
@@ -19,6 +19,11 @@ public function __construct(
     ) {
     }
 
+    public function isConstructor(): bool
+    {
+        return $this->name === '__construct';
```

Comment:
The name of the function registered as the constructor has to be an empty string (`""`).